packages/kbn-optimizer/limits.yml

```diff
@@ -93,7 +93,7 @@ pageLoadAssetSize:
   expressionShape: 34008
   interactiveSetup: 80000
   expressionTagcloud: 27505
-  securitySolution: 273763
+  securitySolution: 301655
```

Comment:
What is being done to address this? The ask to reduce this number was done [here](https://github.com/elastic/kibana/issues/95870). This is loaded on **every** page refresh, affecting the overall experience of Kibana. We have [documentation](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html#ci-metric-resolving-overages) to assist with reducing these bundle sizes through code splitting.

Reply:
@tylersmalley, I wasn't aware of the task you mentioned. After quick look, also found a tech debt task where bundle size should be investigated: https://github.com/elastic/security-team/issues/2441 

Do you consider current limit raise as critical and blocker for this PR and prefer to be handled in it, instead of handling bundle size in dedicated tickets?

Reply:
I was able to resolve it and revert limit increase.
Thanks for bringing attention to this

Reply:
Much appreciated, thanks! ❤️